src/strategies/erc20-balance-of-quadratic-delegation/index.ts

```diff
@@ -0,0 +1,49 @@
+import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
+import { getDelegations } from '../../plugins/delegation/utils';
+
+export const author = 'ferittuncer';
+export const version = '1.0.0';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const delegations = await getDelegations(
+    space,
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
+  );
+  if (Object.keys(delegations).length === 0) return {};
+  console.debug('Delegations', delegations);
```

Comment:
Can we remove the unwanted logs?

Reply:
Actually, this is a `console.debug` not a `console.log` so it will only print when in debugging mode. Do you still think we should remove it? If yes, I'm going to remove when I'm done with debugging.